src/main/archetype/ui.frontend.react.forms.af/src/Form.tsx

```diff
@@ -42,13 +43,23 @@ const Form = (props: any) => {
             setForm(JSON.stringify(json.afModelDefinition))
         }
     }
+    const onSubmit= (action: Action) => {
+      const state = action.target.getState();
+      const thankyouMessage = state?.properties?.['afs:thankyouMessage'];
+      const thankyouPage = state?.properties?.['afs:thankyouPage'];
```

Comment:
use action.payload. It will not be part of properties.

Reply:
also add tests

Reply:
changed

Reply:
There is no setup of the test cases in "ui.frontend.react.forms.af"